packages-backend/loggers/src/loggers.spec.ts

```diff
@@ -43,4 +43,60 @@ describe('application logger', () => {
       )
     );
   });
+  it('should not mutate meta object', () => {
+    // @ts-ignore
+    console._stdout.write = jest.fn();
+    const logger = createApplicationLogger({
+      json: true,
+      formatters: [
+        rewriteFieldsFormatter({
+          fields: [
+            {
+              from: 'meta.req.headers.authorization',
+              to: 'meta.req.headers.authorization',
+              replaceValue: () => '[REDACTED]',
+            },
+            {
+              from: 'meta.req',
+              to: 'meta.req',
+              replaceValue: (value) => {
+                return {
+                  // @ts-expect-error
+                  headers: value.headers,
+                  // @ts-expect-error
+                  headersJsonString: JSON.stringify(value.headers),
+                };
+              },
+            },
+          ],
+        }),
+      ],
+    });
+    const fakeRequest = {
+      headers: {
+        authorization: 'Bearer 123',
+      },
+    };
+    logger.info('Test log', {
+      meta: { req: { headers: fakeRequest.headers } },
+    });
+    expect(fakeRequest).toEqual({
+      headers: {
+        authorization: 'Bearer 123',
+      },
+    });
```

Comment:
This is the proof that the original object is not being mutated (with this fix).